<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TreeFuction</string>
  </property>
  <widget class="QStackedWidget" name="stack">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>941</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="fillLimit">
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>941</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Fill Limit</string>
      </attribute>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>201</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Select written files:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>821</width>
         <height>30</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="selectButton_1">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>50</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>191</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Select catalog:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>821</width>
         <height>30</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="selectButton_2">
       <property name="geometry">
        <rect>
         <x>860</x>
         <y>170</y>
         <width>50</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="fillButton">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>350</y>
         <width>112</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>Fill</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Update Tree</string>
      </attribute>
      <widget class="QTreeWidget" name="oldTreeWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>461</width>
         <height>321</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">Old Tree</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="newTreeWidget">
       <property name="geometry">
        <rect>
         <x>469</x>
         <y>30</y>
         <width>461</width>
         <height>321</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string notr="true">New Tree</string>
        </property>
       </column>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Old Version:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>0</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>New Version:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="oldOpenButton">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>360</y>
         <width>161</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Old File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="newOpenButton">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>360</y>
         <width>161</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>New File</string>
       </property>
      </widget>
      <widget class="QPushButton" name="updateButton">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>360</y>
         <width>161</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
      <widget class="QPushButton" name="updateSaveButton">
       <property name="geometry">
        <rect>
         <x>750</x>
         <y>360</y>
         <width>161</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QPushButton" name="compareButton">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>360</y>
         <width>161</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="assignAddress">
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="assignButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Assign</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>112</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>361</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QListWidget" name="list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>371</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ptyTree</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>AssignAddress</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="quitButton">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>400</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>1081</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
